relay-event-normalization/src/event.rs

```diff
@@ -156,6 +212,18 @@ pub fn normalize_event(event: &mut Annotated<Event>, config: &NormalizationConfi
         let _ =
             trimming::TrimmingProcessor::new().process_event(event, meta, ProcessingState::root());
     }
+
+    if config.is_last_normalize && config.remove_other.unwrap_or(!is_renormalize) {
+        // Remove unknown attributes at every level
+        let _ =
+            remove_other::RemoveOtherProcessor.process_event(event, meta, ProcessingState::root());
```

Comment:
These two processors don't return `ProcessingResult` errors that drop events, so it's safe to ignore the result.

Reply:
Would be good to add this comment to the code

Reply:
I find the comment redundant -- if you're reading it, you already see the result being ignored.